### DISCUSSION TITLE
Mode selection

### DISCUSSION THREAD

**Discussion by cyr666888** [BartoszCichecki/LenovoLegionToolkit#1189] [Ideas]
The drop-down box of performance mode selects whether there are more custom modes

**Comment by BartoszCichecki** [comment]
I have no clue what this means.

**Reply by cyr666888** [reply]
If you choose a custom mode, if you want to optimize for gaming, choose it. But when you use it daily, the built-in balanced mode is not optimal for power consumption, so you have to customize it, but there is only one option.

**Reply by BartoszCichecki** [reply]
Is still don't understand what you want.

**Reply by cpodlewski** [reply]
I guess, if you could give users access to customize Lenovo's proprietary Balanced Mode, this could come in handy, if you could use this customized balanced profile in DC mode. On AC, one can switch to Custom Mode and thanks to Bartosz's great tool add Sub-Profiles.
